android/src/test/java/com/internxt/mobilesdktests/utils/CryptoTest.kt

```diff
@@ -0,0 +1,42 @@
+package com.internxt.mobilesdktests.utils
+
+import com.facebook.common.util.Hex
+import com.internxt.mobilesdk.utils.CryptoUtils
+import org.junit.jupiter.api.Test
+import kotlin.test.assertTrue
+
+internal class CryptoUtilsTest {
+
+  private val SUT = CryptoUtils
+
+  @Test
+  fun `Should list available providers`() {
+    // This is just to print the available algorithms in your JVM, is not asserting anything
+    // and is not a test at all
+    val available = SUT.getAvailableAlgorithms()
+
+    println(available)
```

Comment:
Remove this println

Reply:
This print is intented, this tests should print all the available algorhitms in your JVM, is for debugging purposes but very useful